index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link rel="stylesheet" href="style.css" />
+
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>FlavioContas</title>
+  </head>
+  <body>
+    <header>
+      <h1>Tabuada</h1>
+    </header>
+    <section>
+      <article>
+        <form id="form">
+          <div>
+            <label id="multi">Multiplicador:</label>
+            <input type="number" id="input" />
+            <button id="button">Calcular</button>
```

Comment:
Este não é um simples `button`, é um button de submição de formulário por isso tem de ter a propriedade `type="submit"`

```suggestion
            <button type="submit" id="button"> Calcular</button>
```

https://www.w3schools.com/tags/tag_button.asp